sdk/keyvault/azure-mgmt-keyvault-autorestv5/azure/mgmt/keyvault/v2018_02_14/aio/operations_async/_private_endpoint_connections_operations_async.py

```diff
@@ -119,16 +120,21 @@ def __init__(self, client, config, serializer, deserializer) -> None:
         :param private_endpoint_connection_name: Name of the private endpoint connection associated
          with the key vault.
         :type private_endpoint_connection_name: str
-        :param properties: The intended state of private endpoint connection.
-        :type properties: ~azure.mgmt.keyvault.v2018_02_14.models.PrivateEndpointConnection
+        :param private_endpoint: Properties of the private endpoint object.
```

Comment:
Just want to make sure: when you regenerated this time, did you specify a payload flattening threshold? Did your command change in anyways?

Reply:
> Just want to make sure: when you regenerated this time, did you specify a payload flattening threshold? Did your command change in any ways?

@iscai-msft No, I didn't change my regen command.
```
autorest --python --use=/mnt/c/repos/autorest.python /mnt/c/repos/azure-rest-api-specs/specification/keyvault/resource-manager/readme.md --multiapi --python-sdks-folder=/mnt/c/repos/azure-sdk-for-python/sdk --debug
```